Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -1186,6 +1198,22 @@ bool LocalFrameView::flushCompositingStateIncludingSubframes()
     return allFramesFlushed;
 }
 
+void LocalFrameView::enableRenderingSuppressionStateIncludingSubframes()
```

Comment:
Move this function up to be below `enableRenderingSuppressionStateForThisFrame()`